packages/docs/src/styles/components/navigation.scss

```diff
@@ -133,42 +138,23 @@
   }
 }
 
+.ds-c-vertical-nav__label {
+  &:hover,
+  :visited:hover {
+    --vertical-nav-item__background-color--hover: color-mix(
```

Comment:
all the code in the `:before` styles seemed to be in an attempt to modify an existing color token to be something we don't have (adding an alpha to whatever the vert nav hover bg was). 

css filters are expensive in terms of overall perf, so removing that was a boon to us. also using [color-mix()](https://caniuse.com/?search=color-mix) to accomplish the transparency needs.

side-note: i don't really think we should be combining/modifying tokens for one-off implementation, and the best solution here was probably to update this to just rely on `--color-primary-lightest`. i tried that out, but the color was noticeably different than our current color and decided to be less aggressive with the change. maybe add this to the pile for things to review if we update the docs site?

Reply:
Great analysis and solution! Very elegant

Sounds good to pile it onto future doc site changes 👍